### PR TITLE
Metatranscriptomics tutorial: updates

### PR DESCRIPTION
**Pull request by bebatut**
- Update HUMAnN and MetaPhlAn tools to new version
- Replace some steps by HUMAnN tools
- Add tool ids in hands-on boxes
- Work on text, update results
- Update workflows

ping  @subinamehta and @shiltemann: working on it to be done asap for your training next week

### CODE REVIEW

File: topics/metagenomics/tutorials/metatranscriptomics/content.md [hunk 1]
```diff
@@ -84,7 +81,7 @@ A graphical representation of the ASaiM workflow which we will be using today is
 >    {% snippet faqs/galaxy/histories_create_new.md %}
 >    {% snippet faqs/galaxy/histories_rename.md %}
 >
-> 2. Import `T1A_forward` and `T1A_reverse` from [Zenodo](https://zenodo.org/record/3362849) or from the data library (ask your instructor)
+> 2. [Import](upload1) `T1A_forward` and `T1A_reverse` from [Zenodo]({{ page.zenodo_link }}) or from the data library (ask your instructor)
```
Comment (hexylena) [inline topics/metagenomics/tutorials/metatranscriptomics/content.md:84]:
```suggestion
> 2. {% tool [Import](upload1) %} `T1A_forward` and `T1A_reverse` from [Zenodo]({{ page.zenodo_link }}) or from the data library (ask your instructor)
```

**Comment by subinamehta** [comment]
Thanks @bebatut ...let me know..I will also start testing! Thanks @hexylena for your suggestions!! Thank you both for working so hard!

**Comment by bebatut** [comment]
It is not ready for now (currently fixing some stuff). I hope to finish it by tomorrow.  @subinamehta and @shiltemann I will let you know when ready

**Comment by bebatut** [comment]
@shiltemann and @subinamehta it is finally ready :smile:

**Comment by bebatut** [comment]
I will fix the linting issues tomorrow

File: topics/metagenomics/tutorials/metatranscriptomics/tutorial.bib [hunk 2]
```diff
@@ -1,73 +1,30 @@
...
-  author={Batut, B{\'e}r{\'e}nice and Gravouil, K{\'e}vin and Defois, Cl{\'e}mence and Hiltemann, Saskia and Brug{\`e}re, Jean-Fran{\c{c}}ois and Peyretaillade, Eric and Peyret, Pierre},
-  journal={GigaScience},
-  volume={7},
-  number={6},
-  pages={giy057},
-  year={2018},
-  publisher={Oxford University Press}
-}
 
-@article{franzosa2018species,
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/metatranscriptomics/tutorial.bib:12]: were all these removed intentionally?

Comment (bebatut) [reply topics/metagenomics/tutorials/metatranscriptomics/tutorial.bib:12]: No. I do not know what happened here

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Thanks so much @bebatut!

**Pull request merged**